Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -301,6 +307,14 @@ export function updateOptions(element, elementId, options) {
             picker.set("disable", disableDatesOptions.concat(disableDaysOptions));
         }
 
+        if (options.enabledDates.changed) {
+            if (options.enabledDates.value === null) {
```

Comment:
Check for both null or undefined?

Reply:
I don't think there is a pathway for it, but you never know with js.. so I changed it

Reply:
We should already have these kind of checks in the code base, no?
Might be worth just having a general utility for **isNullOrUndefined** to avoid having multiple ways to check across the js code base.